app/build.gradle.kts

```diff
@@ -146,6 +147,8 @@ dependencies {
     val composeBom = platform(libs.compose.bom)
 
     // Dependencies
+    implementation(libs.kotlinx.serialization.json.v163)
+    implementation(libs.kotlinx.serialization.json.v160)
```

Comment:
Question: Why 2 versions ?

Reply:
I won't change the code itself, let me look into the gradle issues